go-controller/pkg/config/config.go

```diff
@@ -401,6 +401,9 @@ func buildOvnAuth(direction, externalID string, cliAuth, confAuth *rawOvnAuthCon
 	}
 	if address == "" && readAddress {
```

Comment:
It is true, but you also provide the addresses as command line arguments so here address will not be "". It will have those values: https://github.com/shettyg/ovn-kubernetes/blob/dee986c84b6404a0ae98ce36420b80dfd655a85a/vagrant/provisioning/setup-k8s-minion.sh#L68-L69
I saw the error while doing some more development in the Windows CNI which required me to run some OVN commands and then I noticed it.
Theoretically you can see the error if you run gateway-init the second time without passing the OVN addresses since they can be fetched from external_ids.